lib/base/strings.dart

```diff
@@ -297,6 +299,7 @@ abstract class Strings {
 
   // Errors
   static const error = 'Error';
+  static const cantLaunchUrl = 'The app can not launch the Url';
```

Comment:
How about "Couldn't find an app to open $url"?

Reply:
I had the same thought but the URL might not always be meaningful for a user e.g. MobilePay Deep links etc. Then what could make sense would probably be to log the errorful url